### ISSUE TITLE
README ambiguity: System Preparation - Raspberry Pi / ARM64

### ISSUE THREAD

**Issue by alex-matthews** [onedr0p/flux-cluster-template#833]
See inline comments.

#### Raspberry Pi / ARM64

[...]

##### Installing the OS

There is a decent guide [here](https://raspberrytips.com/install-debian-on-raspberry-pi/) on how to get Debian installed on a Raspberry Pi.

1. Deviations from that guide **<<<COMMENT: We should highlight another deviation: Disregard "Adding the desktop environment">>>**, as per [these](https://raspi.debian.net/defaults-and-settings/) instructions. **<<<COMMENT: These aren't really instructions, so I suggest we remove the reference>>>**

    ```txt
    - Use an SSD instead of an SD Card
    **<<<COMMENT: The following matters are supplemental to the guide (I think), so I suggest we make them a post-install item>>>**
    - Reinsert the storage media **<<<QUESTION: What does this mean?>>>**
    - Open `sysconf.txt` file **<<<QUESTION: Located where?>>>**
    - Add `root_authorized_key` **<<<QUESTION: Where does the key come from? Also, does this operation overlap with [How to configure SSH key-based authentication](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) mentioned elsewhere in the README?>>>**
    - Add `root_pw`
    - add `hostname`
    ```

2. (Post install) I would recommend adding a generic user. **<<<COMMENT: The guide already covers this in detail>>>**

   ```
   adduser <username>
   ```

**<<<COMMENT about the System Preparation section generally: We should clarify how it relates to the "Preparing Debian Server with Ansible" section>>>**

Some ideas:
- If item 1 of the Ansible section ([How to configure SSH key-based authentication](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server)) still applies, move it to this section.
- In the Ansible section, state whether the commands are for the root user or the normal/generic user created here.

**Comment by onedr0p** [comment]
CC @LilDrunkenSmurf

**Comment by LilDrunkenSmurf** [comment]
`Disregard "Adding the desktop environment"`
Fair. We can honestly add a comment to disregard anything after `Flash Debian to the SD Card`, as we add extra instructions to make a few changes before the first boot.

`These aren't really instructions, so I suggest we remove the reference`
They're not instructions, but references to the values that we're changing. It depends if we see value in referencing the values, or just stating what to change with no links back to the source. I'm good, either way.

`**<<<COMMENT: The following matters are supplemental to the guide (I think), so I suggest we make them a post-install item>>>**`
Agreed.

`**<<<QUESTION: What does this mean?>>>**`
I did this on a macbook. The RPi imager ejected the disk after verifying the image, so I had to physically unplug and plug the USB back into my macbook to mount the external drive. YMMV with windows, so it may not be the same. We can expand on that.

` **<<<QUESTION: Located where?>>>**`
It was in the root dir of the external drive, when it was mounted.

`**<<<QUESTION: Where does the key come from? Also, does this operation overlap with [How to configure SSH key-based authentication](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) mentioned elsewhere in the README?>>>**`
It totally overlaps. I agree with moving it to this section.

Personally, I only added the generic user as a backup if my SSH keys were lost, I found the guide after the fact, as I was just looking for a pre-written guide on how to flash an external drive with RPi imager. I failed to read how much further it actually goes.
